app/controllers/general.php

```diff
@@ -1257,9 +1307,14 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
             ->addHeader('Pragma', 'no-cache')
             ->setStatusCode($code);
 
-        $template = ($route) ? $route->getLabel('error', null) : null;
+        $template = $error->getView() ?? (($route) ? $route->getLabel('error', null) : null);
+
+        // TODO: Ideally use group 'api' here, but all wildcard routes seem to have 'api' at the moment
```

Comment:
@Meldiron do we wish to resolve this TODO before merging?

Reply:
I tried, no easy way. Dont want to do big refactor as part of thi sPR, so for now good enough